<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditor</class>
 <widget class="QWidget" name="MapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../textedit.qrc">:/images/map.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Total:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbTotal">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewMapButton">
         <property name="toolTip">
          <string>View Map Detail</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../textedit.qrc">
           <normaloff>:/images/ViewDetailMap.png</normaloff>:/images/ViewDetailMap.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addMapButton">
         <property name="toolTip">
          <string>Add Map</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../textedit.qrc">
           <normaloff>:/images/AddMap.png</normaloff>:/images/AddMap.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeMapButton">
         <property name="toolTip">
          <string>Remove Map</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../textedit.qrc">
           <normaloff>:/images/RemoveMap.png</normaloff>:/images/RemoveMap.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="mapTreeView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findMapButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Find Map</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../textedit.qrc">
           <normaloff>:/images/findMap.png</normaloff>:/images/findMap.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mapeditor.qrc"/>
  <include location="../textedit.qrc"/>
  <include location="../textedit.qrc"/>
 </resources>
 <connections/>
</ui>
